src/bucket/FutureBucket.cpp

```diff
@@ -35,6 +35,7 @@ FutureBucket::FutureBucket(Application& app,
     , mInputCurrBucket(curr)
     , mInputSnapBucket(snap)
     , mInputShadowBuckets(shadows)
+    , mProtocolVersion(maxProtocolVersion)
```

Comment:
Just double checking: is mProtocolVersion stored in the FutureBucket just to allow checking assertions?

Reply:
yeah, sanity check.

Reply:
Cool, thanks.